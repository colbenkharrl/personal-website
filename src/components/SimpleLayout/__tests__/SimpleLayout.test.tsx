import { render, screen } from '@testing-library/react';
import * as stories from '../__stories__/SimpleLayout.stories';
import { composeStories } from '@storybook/react';

const { Basic } = composeStories(stories);

it('SimpleLayout matches snapshot (regression test)', () => {
  render(<Basic />);
  expect(screen.getByTestId('Container')).toMatchSnapshot(
    'Full component snapshot.',
  );
});
