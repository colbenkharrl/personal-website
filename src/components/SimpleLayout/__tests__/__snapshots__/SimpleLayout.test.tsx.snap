// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SimpleLayout matches snapshot (regression test): Full component snapshot. 1`] = `
<div
  class="sm:px-8 mt-16 sm:mt-32"
  data-testid="Container"
>
  <div
    class="mx-auto w-full max-w-7xl lg:px-8"
  >
    <div
      class="relative px-4 sm:px-8 lg:px-12"
    >
      <div
        class="mx-auto max-w-2xl lg:max-w-5xl"
      >
        <header
          class="max-w-2xl"
        >
          <h1
            class="text-4xl font-bold tracking-tight text-zinc-800 dark:text-zinc-100 sm:text-4xl"
          >
            Test title.
          </h1>
          <p
            class="mt-6 text-base text-zinc-600 dark:text-zinc-400"
          >
            This is the intro text.
          </p>
        </header>
        <div
          class="mt-16 sm:mt-20"
        >
          Text.
        </div>
      </div>
    </div>
  </div>
</div>
`;
